### ISSUE TITLE
Question: Next chapter link mode

### ISSUE THREAD

**Issue by TheOne320** [Mitch528/WebNovelConverter#11]
How does this mode work? How would I use it?

**Comment by Mitch528** [comment]
The next chapter link mode works by first giving it a starting chapter (instead of a TOC). After that, it finds all the proceeding chapters after the starting chapter by following the "Next Chapter" links.

This mode is useful for when a TOC isn't updated very often, so you don't have to manually add the chapters that aren't listed there.

**Comment by TheOne320** [comment]
Ok, but the mode does not work very well. Every webnovel I tested it on did not give me any results except the first chapter. I will list them the next time I test it.

**Comment by Mitch528** [comment]
It only works on the web novels that have a "Next Chapter" link shown on each chapter. Did the ones you test it on have that?

**Comment by TheOne320** [comment]
yes, I will report them the next time I try.

**Comment by TheOne320** [comment]
https://kntranslation.wordpress.com/help-gooogle-sensei-v1-prologue/
does not work with next chapter link. Does the wording have to be exactly "Next Chapter"? A lot of sites just use "Next".

**Comment by Mitch528** [comment]
Yeah, it was looking for "Next Chapter"

Fixed.

**Comment by TheOne320** [comment]
What does it look for now? I tested the next chapter mode on [grimgar](https://grimgarthetranslation.wordpress.com/level-3/1223-2/chapter-1-myself-natural-talent-and-bittersweetness/) and it followed something wrong and gave share links.

**Comment by Mitch528** [comment]
Give it a try now.

**Comment by TheOne320** [comment]
Yeah that works now. I will post the next time I find something wrong.

Thank you for all the work you do on this.

**Comment by Mitch528** [comment]
Thank you for the bug reports :+1:

**Comment by TheOne320** [comment]
[Here](http://tseirptranslations.blogspot.com/2016/03/is-b5c57.html) this mode also does not work.

**Comment by TheOne320** [comment]
And [this ](http://lightnovelstranslations.com/himekishi-ga-classmate-isekai-cheat-de-dorei-ka-harem/chapter-1-the-slavemancer-and-the-princess-knight/)uses the next chapter title as the link which probably makes it not work (yet).

**Comment by Mitch528** [comment]
The first one is fixed. However, the second one probably isn't doable, since the link doesn't really have an indicator for the program to determine that it's the "next" chapter.

**Comment by TheOne320** [comment]
Crash when putting [this](http://japtem.com/mg-volume-2-chapter-37/) in next chapter mode.

**Comment by Mitch528** [comment]
Fixed as of c845d66629467fcfbfabeacb0747266dc5769875
